site.yml

```diff
@@ -17,6 +56,32 @@
         - splunk.role != 'splunk_search_head_captain'
         - not splunk.upgrade
 
+    # the get_url module is for linux only
+    - name: Check if Post plays need downloaded
+      when:
+        - ansible_post_tasks is defined
+        - ansible_post_tasks is not none
+        - ansible_system is match("Linux")
+      get_url:
```

Comment:
Do we want retries over these pre-/post- plays?

Reply:
Didn't want to delay the startup if they aren't url's.  I'm torn on it, lets wait and see if anyone asks for it.